roles/ceph-install/tasks/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 - include: ./installs/install_on_redhat.yml
   when: ansible_os_family == 'RedHat'
+  tags:
```

Comment:
@erikh 

Instead of adding the tag here, `ceph_install` role should have been added as dependency for `dev` here https://github.com/contiv/ansible/blob/master/roles/dev/meta/main.yml#L14

I remember having added it but looks like it is missing, may be I lost it during merge. So something like this will be one way to do it in `dev` role dependencies:

```
- { role: ceph-install, tags: 'prebake-for-dev' }
```